### PR TITLE
Add or Update NPCs from Bestiary 2 and PFS 2-00

### PR DESCRIPTION
**Pull request by TMunsie**
Corrects some errors in Bestiary 2

### CODE REVIEW

File: packs/data/pathfinder-bestiary-2.db/culdewen.json [hunk 1]
```diff
@@ -682,7 +682,7 @@
                     "value": null
                 },
                 "description": {
-                    "value": "<p>A creature struck by the culdewen's fish hook is skewered, taking @Localize[PF2E.PersistentDamage.Bleed1d4.success] as long as the hook remains stuck in it. The fish hook can be removed if a creature @Compendium[pf2e.actionspf2e.Escape]{Escapes} (DC 25), pulling it free.</p>"
+                    "value": "<p>A creature struck by the culdewen's fish hook is skewered, taking @Localize[PF2E.PersistentDamage.Bleed1d4.success] as long as the hook remains stuck in it. The fish hook can be removed if a creature @Compendium[pf2e.actionspf2e.Escape]{Escapes} (@Check[type:athletics|dc:25]), pulling it free.</p>"
```
Comment (SpartanCPA) [inline packs/data/pathfinder-bestiary-2.db/culdewen.json:685]: Escape is not necessarily Athletics

File: packs/data/pathfinder-bestiary-2.db/korred.json [hunk 2]
```diff
@@ -1098,7 +1172,7 @@
                     "value": null
                 },
                 "description": {
-                    "value": "<p>5 feet.</p>\n<p>The korred's long, animated hair reaches out and interferes with creatures in the area. The korred can select which targets are affected by their animated hair. An affected creature that ends its turn in the emanation must succeed at a @Check[type:reflex|dc:18] save or become @Compendium[pf2e.conditionitems.Clumsy]{Clumsy 1} as long as it remains in the aura.</p>"
+                    "value": "<p>@Compendium[pf2e.bestiary-ability-glossary-srd.Aura]{Aura} @Template[type:emanation|distance:5]{5 feet}</p>\n<p>The korred's long, animated hair reaches out and interferes with creatures in the area. The korred can select which targets are affected by their animated hair. An affected creature that ends its turn in the emanation must succeed at a @Check[type:reflex|dc:18] save or become @Compendium[pf2e.conditionitems.Clumsy]{Clumsy 1} (@Compendium[pf2e.conditionitems.Clumsy]{Clumsy 2} on a critical failure) as long as it remains in the aura.</p>"
```
Comment (SpartanCPA) [inline packs/data/pathfinder-bestiary-2.db/korred.json:1175]:
```suggestion
                    "value": "<p>@Template[type:emanation|distance:5]{5 feet} @Compendium[pf2e.bestiary-ability-glossary-srd.Aura]{Aura}</p>\n<p>The korred's long, animated hair reaches out and interferes with creatures in the area. The korred can select which targets are affected by their animated hair. An affected creature that ends its turn in the emanation must succeed at a @Check[type:reflex|dc:18] save or become @Compendium[pf2e.conditionitems.Clumsy]{Clumsy 1} (@Compendium[pf2e.conditionitems.Clumsy]{Clumsy 2} on a critical failure) as long as it remains in the aura.</p>"
```

File: packs/data/pathfinder-bestiary-2.db/vexgit.json [hunk 3]
```diff
@@ -994,7 +994,20 @@
                 "proficient": {
                     "value": 0
                 },
-                "rules": [],
+                "rules": [
+                    {
+                        "key": "FlatModifier",
+                        "label": "To Craft Traps",
```
Comment (SpartanCPA) [inline packs/data/pathfinder-bestiary-2.db/vexgit.json:1000]:
```suggestion
                        "label": "PF2E.SkillVariant.CraftTrap",
```

**Review by SpartanCPA — CHANGES_REQUESTED** [review]

File: packs/data/pfs-season-2-bestiary.db/mushroom-ring-1-2.json [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+            },
+            "stealth": {
+                "details": "<p>(trained)</p>",
+                "value": 8
+            }
+        },
+        "creatureType": "",
+        "details": {
+            "description": "<p>This <em>mushroom ring</em> forms a portal to the First World, allowing the Thorned Monarch's reinforcements into the Material Plane. When the PCs approach, a creature emerges. Creatures continue to emerge until the mushroom ring is disabled. Disabling the mushroom ring also causes any creatures that have emerged to disappear.</p>",
+            "disable": "<p>@Check[type:thievery|dc:15](trained) or @Check[type:nature|dc:18]&nbsp;to rearrange the mushrooms to disrupt the portal (two actions); three successes are required to disable the portal</p>",
```
Comment (SpartanCPA) [inline packs/data/pfs-season-2-bestiary.db/mushroom-ring-1-2.json:23]:
```suggestion
            "disable": "<p>@Check[type:thievery|dc:15|traits:action:disable-device](trained) or @Check[type:nature|dc:18] to rearrange the mushrooms to disrupt the portal (two actions); three successes are required to disable the portal</p>",
```

File: packs/data/pfs-season-2-bestiary.db/mushroom-ring-3-4.json [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+            },
+            "stealth": {
+                "details": "<p>(trained)</p>",
+                "value": 12
+            }
+        },
+        "creatureType": "",
+        "details": {
+            "description": "<p>This <em>mushroom ring</em> forms a portal to the First World, allowing the Thorned Monarch's reinforcements into the Material Plane. When the PCs approach, a creature emerges. Creatures continue to emerge until the mushroom ring is disabled. Disabling the mushroom ring also causes any creatures that have emerged to disappear.</p>",
+            "disable": "<p>@Check[type:thievery|dc:22](trained) or @Check[type:nature|dc:22] to rearrange the mushrooms to disrupt the portal (two actions) or <em>@Compendium[pf2e.spells-srd.Dispel Magic]{Dispel Magic}</em> (2nd level, counteract DC 20) to counteract the magical properties; three successes are required to disable the portal</p>",
```
Comment (SpartanCPA) [inline packs/data/pfs-season-2-bestiary.db/mushroom-ring-3-4.json:23]:
```suggestion
            "disable": "<p>@Check[type:thievery|dc:22|traits:action:disable-device](trained) or @Check[type:nature|dc:22] to rearrange the mushrooms to disrupt the portal (two actions) or <em>@Compendium[pf2e.spells-srd.Dispel Magic]{Dispel Magic}</em> (2nd level, counteract DC 20) to counteract the magical properties; three successes are required to disable the portal</p>",
```

File: packs/data/pfs-season-2-bestiary.db/mushroom-ring-5-6.json [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+            },
+            "stealth": {
+                "details": "<p>(trained)</p>",
+                "value": 15
+            }
+        },
+        "creatureType": "",
+        "details": {
+            "description": "<p>This <em>mushroom ring</em> forms a portal to the First World, allowing the Thorned Monarch's reinforcements into the Material Plane. When the PCs approach, a creature emerges. Creatures continue to emerge until the mushroom ring is disabled. Disabling the mushroom ring also causes any creatures that have emerged to disappear.</p>",
+            "disable": "<p>@Check[type:thievery|dc:21](expert) or @Check[type:nature|dc:25](trained) to rearrange the mushrooms to disrupt the portal (two actions) or <em>@Compendium[pf2e.spells-srd.Dispel Magic]{Dispel Magic}</em> (3rd level, counteract DC 23) to counteract the magical properties; three successes are required to disable the portal</p>",
```
Comment (SpartanCPA) [inline packs/data/pfs-season-2-bestiary.db/mushroom-ring-5-6.json:23]:
```suggestion
            "disable": "<p>@Check[type:thievery|dc:21|traits:action:disable-device](expert) or @Check[type:nature|dc:25](trained) to rearrange the mushrooms to disrupt the portal (two actions) or <em>@Compendium[pf2e.spells-srd.Dispel Magic]{Dispel Magic}</em> (3rd level, counteract DC 23) to counteract the magical properties; three successes are required to disable the portal</p>",
```

File: packs/data/pfs-season-2-bestiary.db/mushroom-ring-7-8.json [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+            },
+            "stealth": {
+                "details": "<p>(expert)</p>",
+                "value": 18
+            }
+        },
+        "creatureType": "",
+        "details": {
+            "description": "<p>This <em>mushroom ring</em> forms a portal to the First World, allowing the Thorned Monarch's reinforcements into the Material Plane. When the PCs approach, a creature emerges. Creatures continue to emerge until the mushroom ring is disabled. Disabling the mushroom ring also causes any creatures that have emerged to disappear.</p>",
+            "disable": "<p>@Check[type:thievery|dc:24](expert) or @Check[type:nature|dc:28](expert) to rearrange the mushrooms to disrupt the portal (two actions) or <em>@Compendium[pf2e.spells-srd.Dispel Magic]{Dispel Magic}</em> (4th level, counteract DC 26) to counteract the magical properties; three successes are required to disable the portal</p>",
```
Comment (SpartanCPA) [inline packs/data/pfs-season-2-bestiary.db/mushroom-ring-7-8.json:23]:
```suggestion
            "disable": "<p>@Check[type:thievery|dc:24|traits:action:disable-device](expert) or @Check[type:nature|dc:28](expert) to rearrange the mushrooms to disrupt the portal (two actions) or <em>@Compendium[pf2e.spells-srd.Dispel Magic]{Dispel Magic}</em> (4th level, counteract DC 26) to counteract the magical properties; three successes are required to disable the portal</p>",
```

File: packs/data/pfs-season-2-bestiary.db/nuglub-pfs-2-00.json [hunk 8]
```diff
@@ -0,0 +1,1140 @@
...
+            },
+            "ci": [],
+            "di": {
+                "custom": "",
+                "value": []
+            },
+            "dr": [],
+            "dv": [
+                {
+                    "label": "Cold Iron",
```
Comment (SpartanCPA) [inline packs/data/pfs-season-2-bestiary.db/nuglub-pfs-2-00.json:101]:
```suggestion
```

File: packs/data/pfs-season-2-bestiary.db/nuglub-pfs-2-00.json [hunk 9]
```diff
@@ -0,0 +1,1140 @@
...
+                    "value": "defensive"
+                },
+                "actionType": {
+                    "value": "reaction"
+                },
+                "actions": {
+                    "value": null
+                },
+                "description": {
+                    "value": "<p><strong>Trigger</strong> A Medium creature within the nuglub's reach leaves a square during its move action</p>\n<hr />\n<p><strong>Effect</strong> The nuglub lashes out at the triggering creature's knees and tries to knock them @Compendium[pf2e.conditionitems.Prone]{Prone}. The nuglub makes an Acrobatics check against the creature's Reflex DC. On a success, the target falls and lands prone.</p>"
```
Comment (SpartanCPA) [inline packs/data/pfs-season-2-bestiary.db/nuglub-pfs-2-00.json:853]:
```suggestion
                    "value": "<p><strong>Trigger</strong> A Medium creature within the nuglub's reach leaves a square during its move action</p>\n<hr />\n<p><strong>Effect</strong> The nuglub lashes out at the triggering creature's knees and tries to knock them @Compendium[pf2e.conditionitems.Prone]{Prone}. The nuglub makes an @Check[type:acrobatics] check against the creature's Reflex DC. On a success, the target falls and lands prone.</p>"
```

File: packs/data/pfs-season-2-bestiary.db/nuglub-pfs-2-00.json [hunk 10]
```diff
@@ -0,0 +1,1140 @@
...
+                },
+                "traits": {
+                    "custom": "",
+                    "rarity": "common",
+                    "value": []
+                },
+                "variants": {
+                    "0": {
+                        "label": "+7 traps",
+                        "options": "traps"
```
Comment (SpartanCPA) [inline packs/data/pfs-season-2-bestiary.db/nuglub-pfs-2-00.json:1065]:
```suggestion
                                "action:craft:trap"
                            ]
                        },
                        "selector": "crafting",
                        "value": 2
                    }
                ],
                "slug": null,
                "source": {
                    "value": ""
                },
                "traits": {
                    "custom": "",
                    "rarity": "common",
                    "value": []
                },
                "variants": {
                    "0": {
                        "label": "+7 traps",
                        "options": "action:craft:trap"
```

File: packs/data/pfs-season-2-bestiary.db/quickling-pfs-2-00.json [hunk 11]
```diff
@@ -0,0 +1,1467 @@
...
+            },
+            "ci": [],
+            "di": {
+                "custom": "",
+                "value": []
+            },
+            "dr": [],
+            "dv": [
+                {
+                    "label": "Cold Iron",
```
Comment (SpartanCPA) [inline packs/data/pfs-season-2-bestiary.db/quickling-pfs-2-00.json:96]:
```suggestion
```

File: packs/data/pfs-season-2-bestiary.db/quickling-pfs-2-00.json [hunk 12]
```diff
@@ -0,0 +1,1467 @@
...
+                "actions": {
+                    "value": null
+                },
+                "description": {
+                    "value": "<p><strong>Trigger</strong> The quickling is targeted by a Strike</p>\n<hr />\n<p><strong>Effect</strong> The quickling darts aside, gaining a +2 circumstance bonus to AC, then Strides up to half their Speed after the Strike resolves.</p>"
+                },
+                "requirements": {
+                    "value": ""
+                },
+                "rules": [],
```
Comment (SpartanCPA) [inline packs/data/pfs-season-2-bestiary.db/quickling-pfs-2-00.json:1061]:
```suggestion
                "rules": [
            {
                "domain": "ac",
                "key": "RollOption",
                "option": "cant-catch-me",
                "toggleable": true
            },
            {
                "key": "FlatModifier",
                "predicate": {
                    "all": [
                        "cant-catch-me"
                    ]
                },
                "selector": "ac",
                "type": "circumstance",
                "value": 2
            }
        ],
```

File: packs/data/pfs-season-2-bestiary.db/jinkin-pfs-2-00.json [hunk 13]
```diff
@@ -0,0 +1,864 @@
...
+                },
+                "traits": {
+                    "custom": "",
+                    "rarity": "common",
+                    "value": []
+                },
+                "variants": {
+                    "0": {
+                        "label": "+7 traps",
+                        "options": "trap"
```
Comment (SpartanCPA) [inline packs/data/pfs-season-2-bestiary.db/jinkin-pfs-2-00.json:733]:
```suggestion
                                "action:craft:trap"
                            ]
                        },
                        "selector": "crafting",
                        "value": 2
                    }
                ],
                "slug": null,
                "source": {
                    "value": ""
                },
                "traits": {
                    "custom": "",
                    "rarity": "common",
                    "value": []
                },
                "variants": {
                    "0": {
                        "label": "+7 traps",
                        "options": "action:craft:trap"
```

File: packs/data/pfs-season-2-bestiary.db/pixie-pfs-2-00.json [hunk 14]
```diff
@@ -0,0 +1,2021 @@
...
+                    "value": "Pathfinder Core Rulebook"
+                },
+                "spellType": {
+                    "value": "utility"
+                },
+                "sustained": {
+                    "value": false
+                },
+                "target": {
+                    "value": "1 creature"
```
Comment (SpartanCPA) [inline packs/data/pfs-season-2-bestiary.db/pixie-pfs-2-00.json:648]:
```suggestion
                    "value": "self"
```

File: packs/data/pfs-season-2-bestiary.db/pixie-pfs-2-00.json [hunk 15]
```diff
@@ -0,0 +1,2021 @@
...
+                    "value": ""
+                },
+                "prepared": {
+                    "value": ""
+                },
+                "primarycheck": {
+                    "value": ""
+                },
+                "range": {
+                    "value": "touch"
```
Comment (SpartanCPA) [inline packs/data/pfs-season-2-bestiary.db/pixie-pfs-2-00.json:621]:
```suggestion
                    "value": "self"
```

File: packs/data/pfs-season-2-bestiary.db/awakened-megalania.json [hunk 16]
```diff
@@ -0,0 +1,501 @@
...
+                    "value": "offensive"
+                },
+                "actionType": {
+                    "value": "action"
+                },
+                "actions": {
+                    "value": 1
+                },
+                "description": {
+                    "value": "<p>Large, 2d10+7 bludgeoning, Rupture 16</p>\n<hr />\n<p>@Localize[PF2E.NPC.Abilities.Glossary.SwallowWhole]</p>"
```
Comment (SpartanCPA) [inline packs/data/pfs-season-2-bestiary.db/awakened-megalania.json:311]:
```suggestion
                    "value": "<p>Large, [[/r {2d10+7}[bludgeoning]]]{2d10+7 bludgeoning}, Rupture 16</p>\n<hr />\n<p>@Localize[PF2E.NPC.Abilities.Glossary.SwallowWhole]</p>"
```

File: packs/data/pfs-season-2-bestiary.db/qxal-the-thorned-monarch.json [hunk 17]
```diff
@@ -0,0 +1,4771 @@
...
+                    "value": "offensive"
+                },
+                "actionType": {
+                    "value": "action"
+                },
+                "actions": {
+                    "value": 1
+                },
+                "description": {
+                    "value": "<p>Qxal knows how to disguise the manifestations of their spellcasting to catch their foes off guard. If the next action they use is Casting a Spell that has a verbal component, they can remove the verbal component. This makes the spell quieter and allows them to cast it in areas where sound can't carry. Additionally, if Qxal has cover or greater cover against or is @Compendium[pf2e.conditionitems.Concealed]{Concealed} from one or more observers while Casting a Spell, they can attempt a Stealth check against one or more of those observers' Perception DC. If they succeed at their check against an observer's DC, that observer doesn't notice Qxal is casting a spell.</p>"
```
Comment (SpartanCPA) [inline packs/data/pfs-season-2-bestiary.db/qxal-the-thorned-monarch.json:4560]:
```suggestion
                    "value": "<p>Qxal knows how to disguise the manifestations of their spellcasting to catch their foes off guard. If the next action they use is Casting a Spell that has a verbal component, they can remove the verbal component. This makes the spell quieter and allows them to cast it in areas where sound can't carry. Additionally, if Qxal has cover or greater cover against or is @Compendium[pf2e.conditionitems.Concealed]{Concealed} from one or more observers while Casting a Spell, they can attempt a @Check[type:stealth] check against one or more of those observers' Perception DC. If they succeed at their check against an observer's DC, that observer doesn't notice Qxal is casting a spell.</p>"
```

**Review by SpartanCPA — CHANGES_REQUESTED** [review]

File: packs/data/pathfinder-bestiary-2.db/culdewen.json (hunk 1 above)
Comment (TMunsie) [reply packs/data/pathfinder-bestiary-2.db/culdewen.json:685]: It is in this specific one. Check the source.

File: packs/data/pathfinder-bestiary-2.db/army-ant-swarm.json [hunk 18]
```diff
@@ -231,7 +231,7 @@
                 }
             },
             "img": "systems/pf2e/icons/actions/Passive.webp",
-            "name": "Scent",
+            "name": "Scent (imprecise) 30 feet",
```
Comment (TMunsie) [inline packs/data/pathfinder-bestiary-2.db/army-ant-swarm.json:234]:
```suggestion
            "name": "Scent (Imprecise) 30 feet",
```

Comment (SpartanCPA) [inline packs/data/pathfinder-bestiary-2.db/army-ant-swarm.json]:
```suggestion
            "name": "Scent (Imprecise) 30 feet",
```

Comment (TMunsie) [reply packs/data/pathfinder-bestiary-2.db/army-ant-swarm.json]: It should have been in the merge 6 minutes before

**Review by SpartanCPA — APPROVED** [review]

**Pull request merged**